modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/BulkActions.js

```diff
@@ -164,30 +170,34 @@ function BulkActions({
 							</li>
 						</ul>
 
-						<div className="bulk-actions">
-							{bulkActions.map((actionDefinition, i) => (
-								<button
-									className={classNames(
-										'btn btn-monospaced btn-link',
-										i > 0 && 'ml-1'
-									)}
-									key={actionDefinition.label}
-									onClick={() =>
-										handleActionClick(
-											actionDefinition,
-											formId,
-											formName,
-											loadData,
-											namespace,
-											sidePanelId
-										)
-									}
-									type="button"
-								>
-									<ClayIcon symbol={actionDefinition.icon} />
-								</button>
-							))}
-						</div>
+						{showBulkActionsManagementBarActions ? (
+							<div className="bulk-actions">
+								{bulkActions.map((actionDefinition, i) => (
```

Comment:
actionDefinition -> bulkAction

Reply:
Hey @smottal this code belongs to another pr

Reply:
@CrescenzoRegaLF ahahahha, I checked ALL the files :) ---- so it's a @gianmarcobrunialti 's problem